barretenberg/cpp/src/barretenberg/eccvm/eccvm_verifier.cpp

```diff
@@ -199,7 +199,7 @@ template <typename Flavor> bool ECCVMVerifier_<Flavor>::verify_proof(const plonk
         if (commitment.y != 0) {
             batched_commitment_unshifted += commitment * rhos[commitment_idx];
         } else {
-            info("ECCVM Verifier: point at infinity (unshifted)");
+            // info("ECCVM Verifier: point at infinity (unshifted)");
```

Comment:
I think this else branch should just be removed if there is nothing inside of it

Reply:
ditto